core/web/user_controller.go

```diff
@@ -2,11 +2,12 @@ package web
 
 import (
 	"errors"
-	"fmt"
 	"net/http"
 
 	"github.com/gin-gonic/contrib/sessions"
 	"github.com/gin-gonic/gin"
+	"github.com/jackc/pgconn"
+	pkgerrors "github.com/pkg/errors"
```

Comment:
IIRC this package works as a drop-in replacement for std lib `errors`.

Reply:
Ah right, reworked in some commits as it's not imported in this file anymore, but removed the rename 🙂